oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticDocumentMaker.java

```diff
@@ -181,6 +181,10 @@ protected void indexSimilarityBinaries(ElasticDocument doc, PropertyDefinition p
         // TODO : not implemented
         // see https://www.elastic.co/blog/text-similarity-search-with-vectors-in-elasticsearch
         // see https://www.elastic.co/guide/en/elasticsearch/reference/current/dense-vector.html
+
+        doc.addSimField(pd.name, blob);
+
+        // TODO : RERANK
```

Comment:
This TODO I don't understand... I thought we only rerank (in Lucene) when querying?

Reply:
This was being used for brute force reordering by calculating distance b/w feature vectors after results from lucene were returned. Currently we are are using brute force approach in ES itself, so we don't need this reordering for now. However, I have removed this todo comment.